arrow/api.py

```diff
@@ -29,6 +29,19 @@ def get(
     ...  # pragma: no cover
 
 
+@overload
+def get(
+    __arg1: int,
+    __arg2: int,
+    __arg3: int,
```

Comment:
Hey @f0rk thanks for the PR. Does this fix similar type checking problems if there more than 3 `ints`? i.e. `arrow.get(2021, 3, 9, 14, 6)`

Reply:
it does not but I can update the pr to do so